Script/char.dat

```diff
@@ -802,6 +840,61 @@ guard
       "\"Not that I think about it, I was reassigned as an honor guard after that huge screw-up. I wonder if... Nah, it's probably just a coincidence.\"";
     }
   }
+
+  Config EMISSARY;
+  {
+    AttributeBonus = 30;
+    DefaultIntelligence = 20;
+    DefaultWisdom = 20;
+    EyeColor = rgb16(0, 153, 255);
+    SkinColor = rgb16(230, 140, 100);
+    ClothColor = rgb16(128, 128, 128);
+    HeadBitmapPos = 96, 368;
+    Helmet = 0;
+    BodyArmor = STEEL bodyarmor(PLATE_MAIL) { Enchantment = 2; }
+    RightWielded = OCTIRON OCTIRON meleeweapon(BASTARD_SWORD) { Enchantment = 2; }
+    LeftWielded = EBONY_WOOD shield { Enchantment = 2; }
+    RightGauntlet = STEEL gauntlet { Enchantment = 2; }
+    RightBoot = STEEL boot { Enchantment = 2; }
+    DefaultName = "sir Lancelyn";
+    NameSingular = "emissary";
+    PostFix = "of Aslona";
+    CWeaponSkillHits = { 2, 500, 500; }
+    RightSWeaponSkillHits = 200;
+    LeftSWeaponSkillHits = 200;
+    PanicLevel = 50;
+    CanBeGenerated = false;
+    IsNameable = false;
+    IsUnique = true;
+    CanBeCloned = false;
+    IsPolymorphable = false;
+    UndeadVersions = false;
+    TamingDifficulty = 25;
+    AttachedGod = CRUENTUS;
+    FriendlyReplies =
+    {
+      12,
+      "\"I shouldn't be saying this so openly, but Aslona is in dire straits and *needs* any help it can get.\"",
+      "\"I've heard other emissaries were sent from Aslona even to goblin lands to negotiate assistance.\"",
+      "\"After the old king was assassinated and the Rebellion started, Aslona lost much of its former power.\"",
+      "\"I will see the Rebellion crushed and every last rebel executed for their crimes!\"",
+      "\"My king is young and inexperienced, his army only barely holds the rebels back. My only hope is I can return to him successful.\"",
+      "\"Bandits are roving free through the forests of Aslona and you can never be sure which town supports the king and which the Rebellion. What happened to my country?\"",
```

Comment:
Not here, this is not an implication. These are two independent sentences - (1) bandits are roving, (2) you cannot know which town supports whom. Maybe the "and" can be replaced by ";"?